docs/quickstart/create-and-publish-a-package-using-visual-studio-net-framework.md

```diff
@@ -1,162 +1,176 @@
+1. In the resulting list of project templates, select **Class Library (.NET Framework)**, and then select **Next**.
 
-For this walkthrough, however, you won't write any additional code because a class library from the template is sufficient to create a package. Still, if you'd like some functional code for the package, use the following:
+1. In the **Configure your new project** window, enter *AppLogger* for the **Project name**, and then select **Create**.
 
-```cs
-using System;
+1. To ensure the project was created properly, select **Build** > **Build Solution**. The DLL is found within the Debug folder (or Release if you build that configuration instead).
 
-namespace AppLogger
-{
-    public class Logger
-    {
-        public void Log(string text)
-        {
-            Console.WriteLine(text);
-        }
-    }
-}
-```
+1. (Optional) For this quickstart, you don't need to write any additional code for the NuGet package because the template class library is sufficient to create a package. However, if you'd like some functional code for this sample package, include the following code:
 
-> [!Tip]
-> Unless you have a reason to choose otherwise, .NET Standard is the preferred target for NuGet packages, as it provides compatibility with the widest range of consuming projects. See [Create and publish a package using Visual Studio (.NET Standard)](create-and-publish-a-package-using-visual-studio.md).
+   ```csharp
+   namespace AppLogger
+   {
+       public class Logger
+       {
+           public void Log(string text)
+           {
+               Console.WriteLine(text);
+           }
+       }
+   }
+   ```
+
+   Within a real-world NuGet package, you'd likely implement many useful features with which others can build applications. You can also set the target frameworks. For examples, see [UWP](../guides/create-uwp-packages.md) and [Xamarin](../guides/create-packages-for-xamarin.md).
 
 ## Configure project properties for the package
 
-A NuGet package contains a manifest (a `.nuspec` file), that contains relevant metadata such as the package identifier, version number, description, and more. Some of these can be drawn from the project properties directly, which avoids having to separately update them in both the project and the manifest. This section describes where to set the applicable properties.
+A NuGet package is a manifest in a *.nuspec* file that contains relevant metadata, such as the package identifier, version number, and description. Some of this metadata can be drawn from the project properties directly, which avoids having to separately update them in both the project and the manifest. The following steps describe how to set the applicable properties:
```

Comment:
I find the first sentence confusing. A nuget package has extension `.nupkg`, not `.nuspec`. The `.nupkg` is a zip file which contains a manifest file, which has extension `.nuspec`, but the zip/nupkg also contains other files like dlls, content files, build files, etc.

Reply:
@dksimpson Looks like the first sentence needs to be reverted, so the meaning is not changed.